### PR TITLE
Feature all cases page

### PR DESCRIPTION
**Pull request by MaxAlter**
animation arrow

### CODE REVIEW

File: pages/platforms/[id].tsx [hunk 1]
```diff
@@ -97,75 +98,74 @@ const PlatformDetails = ({
...
+                height={525}
+                sizes="100vw"
+                style={{ maxWidth: "100%", height: "auto" }}
+                priority
+              />
+            )}
+          </div>
+          <ul>
+            <h3 className={markdownStyle.headText}>Technology Stack</h3>
+            {network && network.length > 0 && (
```
Comment (zAndrey95) [inline pages/platforms/[id].tsx]: Please, optimize this code

File: pages/smart-contracts-development/development.tsx [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+import AuditCardItem from "../smart-contract-audits/auditsCardItem";
+import styles from "../smart-contract-audits/styles.module.css";
 
 const DevelopContract = () => {
   const DEVELOPMENT_CONTENT_ARRAY =
-    SERVICES_CONTENT_INF0['Smart Contracts Development'];
+    SERVICES_CONTENT_INF0["Smart Contracts Development"];
   return (
-    <PageContainer title={'Smart Contracts Development'} link={'/#Services'}>
+    <PageContainer title={"Smart Contracts Development"} link={"/#Services"} isArrow={true}>
```
Comment (zAndrey95) [inline pages/smart-contracts-development/development.tsx:13]: You use in all PageContainer - isArrow={true}. In one situation false. Maybe you can create default true, when you need false - you change this

File: pages/platforms/[id].tsx (hunk 1 above)
Comment (MaxAlter) [reply pages/platforms/[id].tsx]: done

File: pages/smart-contracts-development/development.tsx (hunk 2 above)
Comment (MaxAlter) [reply pages/smart-contracts-development/development.tsx:13]: decided

**Pull request merged**